server/src/main/java/com/octopus/teamcity/opentelemetry/server/TeamCityBuildListener.java

```diff
@@ -308,9 +320,13 @@ private void createBuildStepSpans(SRunningBuild build, Span buildSpan) {
     }
 
     private void createBlockMessageSpan(BlockLogMessage blockLogMessage, Span buildSpan, Map<String, Span> blockMessageSpanMap, SRunningBuild build) {
+        if (blockLogMessage.getBlockType().equals("$TEST_BLOCK$")) {
+            //we handle these explicitly when we publish tests (in createTestExecutionSpans)
```

Comment:
Didn't we have an external user relying on these?

Reply:
Are you thinking of https://github.com/OctopusDeploy/opentelemetry-teamcity-plugin/pull/147?